### DISCUSSION TITLE
Leiterplatte mit Display

### DISCUSSION THREAD

**Discussion by turrican944** [tbnobody/OpenDTU#660] [Show and tell]
Ich habe auch Mal eine Leiterplatte erstellt.
![PXL_20230304_200217931](https://user-images.githubusercontent.com/70096465/223066160-d0ff518c-7e2f-4863-9470-d8e9f168049c.jpg)
[Leiterplatte ](https://github.com/turrican944/OpenDTU-PCB) 

Die Leiterplatten Daten habe ich bereitgestellt.

**Comment by gizmo2006** [comment]
Hi,
wo kann man die Leiterplatte bestellen,

**Comment by turrican944** [comment]
Moin
Ich habe sie von JLCPCB, andere Leiterplatte Hersteller sollten aber auch
möglich sein. Bei JLC PCB einfach die ZIP Datei hochladen dann sollte das
funktionieren. Die aktuelle V1.1 habe ich zwar schon bestellt aber noch
nicht getestet, aber da habe ich nur den Bestückungsdruck angepasst.

Gruß Florian

gizmo2006 ***@***.***> schrieb am Di., 7. März 2023, 12:59:

**Reply by gizmo2006** [reply]
okay, 
Danke,

**Reply by DrknMonkey** [reply]
> Ich habe auch Mal eine Leiterplatte erstellt. ![PXL_20230304_200217931](https://user-images.githubusercontent.com/70096465/223066160-d0ff518c-7e2f-4863-9470-d8e9f168049c.jpg) [Leiterplatte ](https://github.com/turrican944/OpenDTU-PCB)
> 
> Die Leiterplatten Daten habe ich bereitgestellt.

Moin,

würdest du auch einen ganzen Bausatz verkaufen mit einem Wemos D1 ESP32 für OpenDTU? 

Grüße
Jens

**Comment by EckmarSchmitz** [comment]
Ich hatte mir bei JLCPCB mal 5 Boards bestellt. Nach vielen Nachforschungen und Angeboten war dies bei weiten das günstigste und innerhalb 10 Tagen nach Bestellung bei mir. Ich habe 3 Platinen bestückt und betreibe alle parallel an einem Hoymiles 1500 Inverter.
Mein einziger Kritikpunkt ist, dass der externe 5V Anschluss leicht exotisch ist. Dort hätte man eine normale 2,5/5 Buchse verwenden können oder wenigsten die entsprechende Bohrung vorsehen können. Befestigungslöcher für das Display wäre auch nicht schlecht gewesen, zumal wenn man die Bauteile steckbar machen will.
Trotzdem Danke für den PCB, sieht mit dem Behäuse einfach gut aus.
PS: Habe noch 2 Platinen gegen kleine Gebühr (Platine und Porte) zu vergeben.

**Comment by Harisjems** [comment]
hallo [EckmarSchmitz](https://github.com/EckmarSchmitz)

ich hätte Interesse für eine Platine, kannst du dich bei mir melden bitte, Zweck Datenaustausch und Versand.
mfg

**Reply by oldieman** [reply]
Bitte über private Email melden:  eckmar@shees.de

**Comment by oldieman** [comment]
Ich habe weitere Leiterplatten zur Verfügung - falls jemand Interesse hat, bitte melden.

**Comment by oldieman** [comment]
Hallo Jens,

 

ich habe nur die nackten Leiterplatten hier. Hatte nochmals nachbestellt.

Wenn du noch brauchst, bitte melden.

 

Gruß

Eckmar

 

 

Von: DrknMonkey ***@***.***> 
Gesendet: Mittwoch, 26. April 2023 22:15
An: tbnobody/OpenDTU ***@***.***>
Cc: oldieman ***@***.***>; Comment ***@***.***>
Betreff: Re: [tbnobody/OpenDTU] Leiterplatte mit Display (Discussion #660)

 

Ich habe auch Mal eine Leiterplatte erstellt.  <https://user-images.githubusercontent.com/70096465/223066160-d0ff518c-7e2f-4863-9470-d8e9f168049c.jpg> Leiterplatte  <https://github.com/turrican944/OpenDTU-PCB> 

Die Leiterplatten Daten habe ich bereitgestellt.

Moin,

verkaufst du ganze Bausätze?

Grüße
Jens

**Reply by DrknMonkey** [reply]
Hallo Eckmar,

ich denke ich baue das ganze dann doch lieber in ein größeres Gehäuse.
Trotzdem vielen Dank für das Angebot.

Grüße
Jens

**Reply by oldieman** [reply]
Passt doch genau hinein. Am Besten einen abgewinkelten USB Stecker verwenden.

![229194172-2bf1d65d-2eb2-45f8-ade8-a1e2b734b272](https://user-images.githubusercontent.com/5185366/234994109-968a5210-0c5e-49a8-9adf-74e153f49ac7.jpg)

**Reply by DrknMonkey** [reply]
Hallo Eckmar,

sorry für die späte Rückmeldung.
So war es nicht gemeint mit dem Gehäuse... ich wollte die gesamte DTU in ein größeres Gehäuse packen damit ich da gleich auch noch andere Sachen mit unter bringen kann wie zb. ein AC/DC Netzteil zur Stromversorgung, und der gleichzeitigen Steuerung eines Solid State Relais für den WR Ausgang. Ich finde die Variante von Florian mit dem Gehäuse von Gainta am besten gelöst da das PCB eben mit allem drum und dran da rein passt ohne Kabelsalat etc.. an dieser Stelle also erstmal vielen Dank an Florian für das veröffentlichen der Dateien für das PCB 👍. Und für ein größeres Gehäuse könnte ich auch eine einfache Platine nehmen... aber mittlerweile bin ich doch zum Entschluss gekommen das dies die beste Wahl ist :). Also lieber Eckmar... ist diese E-Mail Adresse von dir noch aktuell: eckmar@shees.de ? Dann würde ich dir eine Mail schreiben :).

Grüße
Jens

**Comment by EckmarSchmitz** [comment]
Hallo Jens,

 

die Email Adresse ist aktuell.

 

Gruß

Eckmar

 

 

Von: DrknMonkey ***@***.***> 
Gesendet: Samstag, 29. April 2023 23:39
An: tbnobody/OpenDTU ***@***.***>
Cc: Eckmar Schmitz ***@***.***>; Comment ***@***.***>
Betreff: Re: [tbnobody/OpenDTU] Leiterplatte mit Display (Discussion #660)

 

Hallo Eckmar,

sorry für die späte Rückmeldung.
So war es nicht gemeint mit dem Gehäuse... ich wollte die gesamte DTU in ein größeres Gehäuse packen damit ich da gleich auch noch andere Sachen mit unter bringen kann wie zb. ein AC/DC Netzteil zur Stromversorgung, und der gleichzeitigen Steuerung eines Solid State Relais für den WR Ausgang. Ich finde die Variante von Florian mit dem Gehäuse von Gainta am besten gelöst da das PCB eben mit allem drum und dran da rein passt ohne Kabelsalat etc.. an dieser Stelle also erstmal vielen Dank an Florian für das veröffentlichen der Dateien für das PCB 👍. Und für ein größeres Gehäuse könnte ich auch eine einfache Platine nehmen... aber mittlerweile bin ich doch zum Entschluss gekommen das dies die beste Wahl ist :). Also lieber Eckmar... ist diese E-Mail Adresse von dir noch aktuell: ***@***.*** ***@***.***>  ? Dann würde ich dir eine Mail schreiben :).

Grüße
Jens

**Reply by oldieman** [reply]
Hallo Jens,
die Email-Adresse ist aktuell.
Gruß
Eckmar

**Reply by DrknMonkey** [reply]
Hallo Eckmar,

ich habe bereits zwei Platinen von dir bekommen :).
Danke nochmal ;)

Grüße
Jens

**Comment by derandi80** [comment]
Hallo @EckmarSchmitz,

hättest du evtl. 1-2 Platinen für OpenDTU und würdest sie verkaufen? Viele Grüße
Andreas

**Comment by turrican944** [comment]
Moin
Löcher für das Display hätte man machen können, ich hatte sie immer verlötet. (Eagledaten sind ja mit online kann ja auch gerne jemand ändern oder wenn ich Mal ganz viel Lust habe kann ich hat das ja mal machen, weil ich will das ungerne im Layout ein bauen ohne es zu testen und da noch Leiterplatten da sind besteht bei mir zurzeit kein Bedarf neue zu bestellen weil ich auf kommerziellen Verkauf auch keine Lust habe).
Stecker ist nicht exotisch Phoenix Mini-Combicon, da ich den für viele bastel Projekte nutze habe ich den genommen, bislang aber alle DTUs die ich gebaut habe mit USB angeschlossen. Die Beschriftung mit den +5V ist etwas ungünstig platziert, gebe ich zu.

@derandi80 ich hab noch Platinen liegen, kannst welche haben, schick mir ne Mail. (Dg5hfs(ät)jufu.org)

**Comment by EckmarSchmitz** [comment]
Hallo Andreas,

ich habe auch noch Platinen vorrätig.
Bitte melden unter: ***@***.******@***.***>

Gruß


Von: derandi80 ***@***.***>
Gesendet: Donnerstag, 8. Juni 2023 11:51
An: tbnobody/OpenDTU ***@***.***>
Cc: Eckmar Schmitz ***@***.***>; Mention ***@***.***>
Betreff: Re: [tbnobody/OpenDTU] Leiterplatte mit Display (Discussion #660)


Hallo @EckmarSchmitz<https://github.com/EckmarSchmitz>,

hättest du evtl. 1-2 Platinen für OpenDTU und würdest sie verkaufen? Viele Grüße
Andreas
